personality/ui/rpms/config.pan

```diff
@@ -7,7 +7,11 @@ unique template personality/ui/rpms/config;
 variable HEP_OSLIBS ?= true;
 "/software/packages" = {
     if (is_boolean(HEP_OSLIBS) && HEP_OSLIBS) {
-        SELF[escape('HEP_OSlibs_SL6')] = nlist();
+        if (OS_VERSION_PARAMS['major'] == 'sl5') {
+            SELF[escape('HEP_OSlibs_SL5')] = nlist();
+        } else {
```

Comment:
Is `if sl5` ... `else sl6` safe and future proof? There is already a HEP_OSlibs_SL7.

You could do:
```pan
    "/software/packages" = pkg_repl(format('HEP_OSlibs_%s', to_uppercase(OS_VERSION_PARAMS['major'])));
```

Or perhaps:
```pan
variable hep_oslibs_mapping = dict(
    'sl5', 'HEP_OSlibs_SL5',
    'sl6', 'HEP_OSlibs_SL6',
    'sl7', 'HEP_OSlibs_SL7',
);

"/software/packages" = pkg_repl(hep_oslibs_mapping[OS_VERSION_PARAMS['major']]);
```

And maybe throw a nice error for other OS versions?